### PR TITLE
[RAILS-88] Jenkins Job skeleton

### PR DESCRIPTION
**Pull request by zachar**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -33,72 +33,106 @@ services:
...
-      - init
-    environment:
-      << : *keycloak-enviroment
-    command: "-Dkeycloak.migration.action=export -Dkeycloak.migration.realmName=stuart -Dkeycloak.migration.provider=dir -Dkeycloak.migration.usersExportStrategy=SKIP -Dkeycloak.migration.dir=/exported_configs"
-    volumes:
-      - "./exported_configs/.current:/exported_configs"
-    depends_on:
-      - postgres
-    networks:
-      - dev
```
Comment (BuonOmo) [inline docker-compose.yml:56]: I think this one is an error!

**Review by BuonOmo — APPROVED** [review]

**Comment by BuonOmo** [comment]
So nice to have that CI finally working!

File: docker-compose.yml (hunk 1 above)
Comment (GermanDZ) [reply docker-compose.yml:56]: Yes, please don't remove the `keycloak-export` (yet)

File: docker-compose.yml [hunk 2]
```diff
@@ -33,72 +33,106 @@ services:
...
+    image: quay.io/keycloak/keycloak:9.0.3
     environment:
       << : *keycloak-enviroment
-      KEYCLOAK_IMPORT: "/exported_configs/realm-export-full.json"
-    command: "-Dkeycloak.profile.feature.scripts=enabled -Dkeycloak.profile.feature.upload_scripts=enabled"
+      JDBC_PARAMS: "useSSL=false"
+      KEYCLOAK_IMPORT: "/app/init_config/.current/realm-export-base.json"
+    command: "-Dkeycloak.profile.feature.upload_scripts=enabled"
     volumes:
-      - "./exported_configs/.current:/exported_configs"
```
Comment (GermanDZ) [inline docker-compose.yml:91]: this is gonna break the init/upgrade/export tasks, but don't worry about it for now.

**Review by GermanDZ — APPROVED** [review]
Go, iterate later! 


🌮 🌮  for putting the ✅ on Jenkins! 👏

File: docker-compose.yml (hunk 1 above)
Comment (zachar) [reply docker-compose.yml:56]: I haven't meant to delete it, it happened while I was rebasing.

File: docker-compose.yml (hunk 2 above)
Comment (BuonOmo) [reply docker-compose.yml:91]: @GermanDZ it won't I'm working on it!

**Pull request merged**